examples/ensemble/plot_stack_predictors.py

```diff
@@ -244,27 +242,38 @@ def plot_regression_results(ax, y_true, y_pred, title, scores, elapsed_time):
 fig, axs = plt.subplots(2, 2, figsize=(9, 7))
 axs = np.ravel(axs)
 
+splits = namedtuple("splits", ("train", "test"))
+cv = [splits(*data) for data in KFold(n_splits=5).split(X, y)]
```

Comment:
You don't need to create a `namedtuple`. Using only 

```python
cv = KFold(n_splits=5, shuffle=False)
```

would be enough. It will provide deterministic splits.